src/langrila/memory/cosmos.py

```diff
@@ -0,0 +1,45 @@
+from azure.cosmos import exceptions, CosmosClient, PartitionKey
+import logging
+import os
+import uuid
+
+from ..base import BaseConversationMemory
+
+class CosmosConversationMemory(BaseConversationMemory):
+    def __init__(self, endpoint_name: str, key_name: str, dbname_name: str, containername_name: str, pkey_name: str):
```

Comment:
```suggestion
def __init__(self, endpoint_name: str, key_name: str, dbname_name: str, container_name: str):
```

Reply:
It's a little hard to understand that the name of these arguments mean the name of env variables. But I can't judge what is the best way to fix. Because I'd like to align the naming rule that you use in other modules.
Are you acceptable a rule as follows?
endpoint_name -> endpoint_env_name, key_name -> key_anv_name,...

Reply:
> It's a little hard to understand that the name of these arguments mean the name of env variables. 

You're exactly right. It is problem of my rough coding design.

> endpoint_name -> endpoint_env_name, key_name -> key_anv_name,...

Sure, I feel it's better rule for readability. Thanks! Also I will change argument names of other env variables used at other modules like from `api_key_name` to `api_key_env_name`.